src/api/lib/delayed_job_monitor.rb

```diff
@@ -0,0 +1,16 @@
+class DelayedJobMonitor
+  def run
+    send_email if Delayed::Job.count >= delayed_jobs_warning_limit
```

Comment:
I think we should convert delayed_jobs_warning_limit to an integer. In case that this option is not set.